lib/mutant/mutator/registry.rb

```diff
@@ -21,22 +22,30 @@ def initialize
       # @return [self]
       def register(type, mutator)
         fail RegistryError, "Invalid type registration: #{type}" unless AST::Types::ALL.include?(type)
-        fail RegistryError, "Duplicate type registration: #{type}" if @registry.key?(type)
-        @registry[type] = mutator
+        fail RegistryError, "Duplicate type registration: #{type}" if contents.key?(type)
+        contents[type] = mutator
         self
       end
 
+      # Call registry
+      #
+      # @return [Enumerable<Mutation>]
+      def call(node, parent = nil)
+        lookup(node.type).call(node, parent)
+      end
+
+    private
+
       # Lookup mutator class for node
       #
-      # @param [Parser::AST::Node] node
+      # @param [Symbol] type
       #
       # @return [Class]
       #
       # @raise [ArgumentError]
       #   raises argument error when mutator class cannot be found
-      def lookup(node)
-        type = node.type
-        @registry.fetch(type) do
+      def lookup(type)
+        contents.fetch(type) do
           fail RegistryError, "No mutator to handle: #{type.inspect}"
```

Comment:
Why use `type.inspect` in the error here, but not above? I would suggest choosing one approach in error messages.